langchain/src/util/tests/openai-stream.test.ts

```diff
@@ -131,3 +141,30 @@ describe("OpenAI Stream Tests", () => {
     expect(receivedChunks).toEqual([]);
   });
 });
+
+describe("Azure OpenAI Stream Tests", () => {
+  it("should return a 200 response chunk by chunk", async () => {
+    // When stream mode enabled, Azure OpenAI responds with chunks without tailing blank line.
+    // In addition, Azure sends chunks in batch.
```

Comment:
OpenAI sends data line by line:

https://github.com/hwchase17/langchainjs/assets/96157/5bd0173f-6aff-4e38-a9b4-ce09e07d2cbc

Azure OpenAI sends in batch:

https://github.com/hwchase17/langchainjs/assets/96157/7d7b171e-7d26-4f15-9a13-deb91cd2f62b